packages/site-docs/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "blueprintjs.com/docs",
+  "name": "@blueprintjs/site-docs",
```

Comment:
idea behind the previous name was that it's obvious that the package is not published. I'm sure you ran into the issue where `/` is an invalid char in yarn package names. how about `blueprintjs.com-docs`?

Reply:
I find it weird that this monorepo doesn't follow the standard of using a single NPM scope. `"private": true` should make that publishing fact obvious enough.